schema/deploy/database_functions/correct_gwp_values.sql

```diff
@@ -0,0 +1,113 @@
+-- Deploy ggircs-portal:database_functions/correct_gwp_values to pg
+
+/**
+  Some applications during the 2023 reporting cycle (2022 reporting year) were created before the correct gwp
+  values were applied, which means when the applications were initialized the old AR4 values were applied.
+  This function updates the gwp values in place to the correct AR5 values.
+**/
+
+begin;
+
+create or replace function ggircs_portal_private.correct_gwp_values()
+returns void as $$
+
+  -- CH4
+update ggircs_portal.form_result
+  set form_result = replace(form_result::text, '"gwp": 25', '"gwp": 28')::jsonb
+  where application_id in (
+    select application_id
+      from ggircs_portal.application_revision ar
+      join ggircs_portal.application a
+      on a.id = ar.application_id
+      and a.reporting_year = 2022
+      and ar.created_at < '2023-05-30' and ar.created_at > '2023-04-01'
+  )
+  and form_id=2;
```

Comment:
what's `form_id`?

Reply:
form_id is a foreign key to the form_json table. The form_json table contains the json schemas for each form. In this case, form_id=2 is the emissions schema. So this condition ensures we're only touching emission form data (and not fuel, production or admin data)